src/macaron/policy_engine/examples/aggregate.dl

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved. */
+/* Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/. */
+
+
+#include "prelude.dl"
+
+/**
+ * This is an example of applying the aggregate level policy.
+ */
+
+apply_policy_to("aggregate_level_3", repo) :-
+    // if we have provenance, then require using a trusted builder,
+    // verifying the provenance attestations, and
+    // dependencies must use some kind of scripted build
+    repository(repo, name,_,_,_,_,_),
```

Comment:
```suggestion
    repository(repo, name, _, _, _, _, _),
```

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/46/commits/73c1c81b6bdb6c999112532dab89103685eca48b